README.md

```diff
@@ -56,23 +48,19 @@ https://annu12340.github.io/Javascript-Notes/
 		
   ### Negative INFINITY
   
-        NEGATIVE_INFINITY is displayed when a number exceeds the lower limit of the floating 
+               NEGATIVE_INFINITY is displayed when a number exceeds the lower limit of the floating 
 		point numbers, which is **-1.797693134862316E+308.**
 		
-  
-  **POSITIVE_INFINITY** vs. **MAX_VALUE** 
-  The value of the MAX_VALUE property is the largest number 
-  your JavaScript interpreter can handle. Larger value will be viewed as POSITIVE_INFINITY.
-
-  
-  **NEGATIVE_INFINITY** vs. **MIN_VALUE** 
-  The value of the MIN_VALUE property is the smallest 
-  (closest to zero) number your JavaScript interpreter can handle, while NEGATIVE_INFINITY is 
-  the largest negative number the JavaScript interpreter can represent.
-  The value of NEGATIVE_INFINITY and POSITIVE_INFINITY are read-only : they cannot be changed by 
-  your scripts; they are returned by JavaScript whenever a function or operation returns a number 
-  larger than the MAX_VALUE the JavaScript interpreter can handle.
-  
+  **POSITIVE_INFINITY** vs. **MAX_VALUE** (or **NEGATIVE_INFINITY** vs. **MIN_VALUE** )
+   1. The value of the MAX_VALUE property is the largest number your JavaScript interpreter can             handle. Larger value will be viewed as POSITIVE_INFINITY.
+   2. The value of NEGATIVE_INFINITY and POSITIVE_INFINITY are read-only they cannot be changed by
+      your scripts; they are returned by JavaScript whenever a function or operation returns a number
+      larger than the MAX_VALUE the JavaScript interpreter can handle
+		**See the code snippet**
+		
+         ![Code](https://raw.githubusercontent.com/sumitt1080/demo/master/Capture1.jpg)
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------	 
+	 
   The script below illustrate how the JavaScript interpreter understands the concept of infinity (anything out of bound is infinite).
 
  
```

Comment:
Content-wise everything looks good
Could you add this topic under 'Timer function' section

Reply:
Sure